ui/Screen/Profile/Tracking.svelte

```diff
@@ -34,9 +34,27 @@
   ];
 
   const untracked = [
-    { urn: "hwd1yreg4khbjfa4gsyrio3f7ehluwkdhyregs4k" },
-    { urn: "fjkldasjfkdlsajfio943we859043ikjioclesdjf" },
-    { urn: "fjkldasjfkdlsajfio39409340390we859043ikjioclesdjf" },
+    {
+      urn: "hwd1yreg4khbjfa4gsyrio3f7ehluwkdhyregs4k",
```

Comment:
We should use the URN format that we settled on in: https://github.com/radicle-dev/radicle-upstream/issues/840.

```suggestion
      urn: "@hyndb5gs95gwtsf37tncz4ag3wqrg4ejw3qqga6x1srw9jp8jw59d6.git",
```

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/906/commits/817faffb393cc5d50d70605e357d87a6cb7dc544